src/KeyboardLayout_pt_BR.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * Portuguese keyboard layout.
+ */
+
+#include "KeyboardLayout.h"
+
+extern const uint8_t KeyboardLayout_pt_BR[128] PROGMEM =
+{
+	0x00,          // NUL
+	0x00,          // SOH
+	0x00,          // STX
+	0x00,          // ETX
+	0x00,          // EOT
+	0x00,          // ENQ
+	0x00,          // ACK
+	0x00,          // BEL
+
```

Comment:
For consistency with the other layouts, it would be nice to keep the same layout of the file, without empty lines except after the control characters.